internal/viridian/cluster.go

```diff
@@ -115,17 +130,21 @@ func (a API) GetCluster(ctx context.Context, idOrName string) (Cluster, error) {
 	if err != nil {
 		return Cluster{}, err
 	}
-	c, err := doGet[Cluster](ctx, fmt.Sprintf("/cluster/%s", cluster.ID), a.Token())
+	c, err := WithRetry(ctx, a, func() (any, error) {
+		return doGet[Cluster](ctx, fmt.Sprintf("/cluster/%s", cluster.ID), a)
+	})
 	if err != nil {
 		return Cluster{}, fmt.Errorf("retrieving cluster: %w", err)
 	}
-	return c, nil
+	return c.(Cluster), nil
 }
 
 func (a API) ListClusterTypes(ctx context.Context) ([]ClusterType, error) {
-	csw, err := doGet[Wrapper[[]ClusterType]](ctx, "/cluster_types", a.Token())
+	csw, err := WithRetry(ctx, a, func() (any, error) {
```

Comment:
Same comment with other `WithRetry` code.

Reply:
Fixed in: https://github.com/hazelcast/hazelcast-commandline-client/pull/300/commits/d3d3b8eb6da57e1de512a87e7ebd5597a1c31c6f